package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "client",
-  "version": "0.14.1",
+  "version": "1.1.1",
```

Comment:
Ideally we should set this back to the correct version so we can track changes. So now you can use your script to bump the version to 0.14.2. Learn more about semantic versioning here: https://semver.org/